Makefile

```diff
@@ -49,14 +49,15 @@ PLT_SETUP_OPTIONS =
 SRC_CATALOG = $(DEFAULT_SRC_CATALOG)
 
 CPUS = 
+MAXLOAD = "-1.0"
 
 in-place:
 	if [ "$(CPUS)" = "" ] ; \
```

Comment:
Should this test for the `MAXLOAD` variable? Or if not, the INSTALL.txt documentation should say that `MAXLOAD` does nothing without `CPUS`.

Reply:
`MAXLOAD=-1.0` is the default i.e. no maximum - I set it like this so I don't need to test it here.

Reply:
My point is that `make MAXLOAD=12` will not take the true branch of that conditional, and thus will not pass `--load-average` at all.

Reply:
Ah, right. Well spotted. OK, thanks for the comments. Will need to revisit the patch.